src/pages/Swap/util.js

```diff
@@ -0,0 +1,46 @@
+// 获取路由中某个参数
+const getQueryVariable = variable => {
+  const query = window.location.href.split('?')[1]
```

Comment:
we never access window.location directly, we always use the react router provided location